usaspending_api/api_docs/api_documentation/README.md

```diff
@@ -0,0 +1,28 @@
+
+# USASpending API Documentation
+
+
+The Data Accountability Transparency Act (DATA) Act created the USASpending API to advance the transparency of the Federal Government's spending data.  This API's primary use cases are to serve the USASpending.gov website, as well as US citizens as open API. This API's goal is to serve up to date spending data in a quick, accurate manner.
+
+
+
+The USASpending API is in *V2*. V1 endpoints are currently DEPRECIATED. 
```

Comment:
should be `deprecated`, not `DEPRECIATED`.

Reply:
done